### PR TITLE
Fix function names for mu4e 1.6

### PR DESCRIPTION
**Pull request by seblemaguer**
Some functions have changed names in mu4e 1.6. This PR aims to fix the names of these functions in nano-modeline.

I need to confirm that I didn't forget anything to move the PR from draft to ready

### CODE REVIEW

File: nano-modeline.el [hunk 1]
```diff
@@ -203,7 +203,7 @@ Modeline is composed as:
   (nano-modeline-compose (nano-modeline-status)
                          "Mail"
                          (nano-modeline-mu4e-context)
-                         (format "%d messages" (plist-get mu4e~server-props :doccount))))
+                         (format "%d messages" (plist-get mu4e-server-props :doccount))))
```
Comment (rougier) [inline nano-modeline.el:206]: Maybe we should have intermediate aliasing testing which of the two functions is present and use it, no?

Comment (seblemaguer) [reply nano-modeline.el:206]: yup that is a good idea :)

**Comment by seblemaguer** [comment]
Hopefully, I didn't forget anything :D

**Comment by rougier** [comment]
Thanks and sorry for the delay.

**Pull request merged**